Sources/SnapshotPreviewsCore/ExpandingViewController.swift

```diff
@@ -109,6 +113,13 @@ public final class ExpandingViewController: UIHostingController<EmergeModifierVi
       return
     }
 
+    // Check if we've made too many adjustments
+    if adjustmentCount >= maxAdjustments {
+        print("Warning: Max adjustments reached. Terminating height updates.")
+        runCallback()
+        return
+    }
```

Comment:
@noahsmartin you mentioned over DMs: “We could even report it directly as a warning on that snapshot to indicate which ones had issues with expansion”

is there an example of us currently doing that for any cases?